src/main/java/net/sf/jabref/gui/help/AboutDialogViewModel.java

```diff
@@ -0,0 +1,146 @@
+/*  Copyright (C) 2003-2015 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.gui.help;
+
+import java.io.IOException;
+import javax.inject.Inject;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import javafx.beans.property.ReadOnlyStringProperty;
+import javafx.beans.property.ReadOnlyStringWrapper;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.stage.Stage;
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRef;
+import net.sf.jabref.gui.ClipBoardManager;
+import net.sf.jabref.gui.desktop.JabRefDesktop;
+import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.logic.util.BuildInfo;
+
+public class AboutDialogViewModel {
+
+    private final Log logger = LogFactory.getLog(AboutDialogViewModel.class);
+    private final ReadOnlyStringWrapper website = new ReadOnlyStringWrapper();
+    private final ReadOnlyStringWrapper websiteLibraries = new ReadOnlyStringWrapper();
+    private final ReadOnlyStringWrapper websiteGithub = new ReadOnlyStringWrapper();
+    private final ReadOnlyStringWrapper license = new ReadOnlyStringWrapper();
+    private final ReadOnlyStringWrapper header = new ReadOnlyStringWrapper();
+    private final ReadOnlyStringWrapper years = new ReadOnlyStringWrapper();
+    private final ReadOnlyStringWrapper authors = new ReadOnlyStringWrapper();
+    private final ReadOnlyStringWrapper developers = new ReadOnlyStringWrapper();
+    @Inject
+    protected BuildInfo buildInfo;
+    @FXML
+    private Button closeButton;
+
+    @FXML
+    private void initialize() {
+
+        header.set(String.format("JabRef %s", Globals.BUILD_INFO.getVersion()));

```

Comment:
Can you please check if Globals.Build_Info is used somewhere else? If not then this global variable can be removed and its initialization moved here.